vid/CHANGELOG.md

```diff
@@ -6,3 +6,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## 0.1.0
 
 - Initial release. Verifiable information dispersal trait definition and implementations.
+
+## 0.2.0
+
+- [#650](https://github.com/EspressoSystems/jellyfish/pull/650) perf: VID ADVZ verify_share use parallelism over multiplicity
```

Comment:
nit: 

- put newer version on top (0.2.0 before 0.1.0), reverse chronological 
- remove `feat/feat/..` prefix, those are for commit naming, here we just succinctly describe the changes
- i'd personally divide these into "breaking", "bugs", "non-breaking" such as [this](https://github.com/EspressoSystems/jellyfish/blob/main/plonk/CHANGELOG.md#breaking-changes-1); our [older version](https://github.com/EspressoSystems/jellyfish/blob/main/CHANGELOG_OLD.md#added) is even more granular, but maybe unnecessary

Reply:
done in https://github.com/EspressoSystems/jellyfish/pull/680/commits/93c0ade3d1ee1df0aff364683be549982e4926b6